### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by Ghostuv7** [termux/termux-packages#15384] [Q&A] [answered]
How i can download encrypted data in termux

**Comment by sylirre** [comment, marked as answer]
In general, the data can be downloaded using `curl` or `wget`. Here are examples:

* `curl -LO https://example.com/file.zip`
* `wget https://example.com/file.zip`

Besides the mentioned curl/wget we have other tools, for example `aria2`.

The file can be plain or encrypted, internal contents actually don't matter to download utility.
